packages/create-bison-app/template/package.json.ejs

```diff
@@ -30,14 +31,14 @@
     "g:test:factory": "hygen test factory --name",
     "g:test:request": "hygen test request --name",
     "g:test:util": "hygen test util --name",
-    "generate": "yarn build:prisma && yarn build:nexus && yarn codegen",
     "lint": "yarn eslint . --ext .ts,.tsx --fix --ignore-pattern tmp",
     "start": "next start -p $PORT",
     "test": "yarn withEnv:test jest --runInBand --watch",
     "test:ci": "yarn withEnv:test jest --runInBand",
     "test:e2e": "cross-env DOTENV_CONFIG_PATH=.env.test ts-node -r dotenv/config $(yarn bin)/start-server-and-test 'yarn test:server' http://localhost:3001 'yarn cypress:run'",
     "test:e2e:local": "cross-env CYPRESS_LOCAL=true CYPRESS_BASE_URL=http://localhost:3000 cypress open",
     "test:server": "next start --port 3001",
+    "setup:dev": "yarn db:setup && yarn build:types",
```

Comment:
Just checking if we try to order the scripts alphabetically? It seems like they are, at least everything before colons.

Reply:
A B C D E F G H I J K L M N O P Q R `S`